controllers/control/jobs.go

```diff
@@ -3,22 +3,27 @@ package control
 import (
 	"context"
 	"fmt"
+	"sort"
+	"time"
 
 	"github.com/k8ssandra/cass-operator/pkg/httphelper"
 	"github.com/k8ssandra/cass-operator/pkg/utils"
+	appsv1 "k8s.io/api/apps/v1"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/types"
+	ctrl "sigs.k8s.io/controller-runtime"
 
 	cassapi "github.com/k8ssandra/cass-operator/apis/cassandra/v1beta1"
+	api "github.com/k8ssandra/cass-operator/apis/control/v1alpha1"
 )
 
 // Cleanup functionality
 
 func callCleanup(nodeMgmtClient httphelper.NodeMgmtClient, pod *corev1.Pod, taskConfig *TaskConfiguration) (string, error) {
 	// TODO Add more arguments configurations
 	keyspaceName := ""
-	if keyspace, found := taskConfig.Arguments["keyspace_name"]; found {
+	if keyspace, found := taskConfig.Arguments[api.KeyspaceArgument]; found {
```

Comment:
How is this related to a restart?

Reply:
This is mostly superceded by #395 , just to ensure we don't have magic strings around the code. Same for previous comment.